src/UI/controllers/NativeUIController.js

```diff
@@ -104,13 +104,35 @@ export class NativeUIController extends UIController {
             });
     }
 
+    /**
+     * @param {import('./UIController').AttachKeyboardArgs} args
+     */
+    async attachKeyboard(args) {
+        const { device, form } = args;
+        try {
+            const resp = await device.deviceApi.request(new GetAutofillDataFocusCall(null));
```

Comment:
Shouldn't we pass info about the input type here, so that the native side knows whether to show the accessory or not?